presto-main/src/main/java/com/facebook/presto/sql/gen/SpecialFormBytecodeGenerator.java

```diff
@@ -22,9 +21,9 @@
 import java.util.List;
 import java.util.Optional;
 
-public interface BytecodeGenerator
```

Comment:
Checked the original code; `BytecodeGenerator` is only used by today's special form?

Reply:
Yes, which is why it's renamed to `SpecialFormBytecodeGenerator`. It used to be also used by `FunctionCallCodeGenerator` but now "special form" is separated from "call", `FunctionCallCodeGenerator` no longer implements this.